package.json

```diff
@@ -36,26 +37,29 @@
     "security"
   ],
   "dependencies": {
-    "lodash": "^4.17.15"
+    "lodash": "^4.17.20"
   },
   "devDependencies": {
-    "@serverless/enterprise-plugin": "^1.3.10",
-    "@types/chai": "^4.2.0",
-    "@types/lodash": "^4.14.138",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^6.14.7",
+    "@serverless/enterprise-plugin": "^4.1.1",
+    "@types/chai": "^4.2.14",
+    "@types/lodash": "^4.14.164",
+    "@types/mocha": "^8.0.3",
+    "@types/node": "^14.14.6",
```

Comment:
looks like we should leave it at 12 for now

Reply:
dependencies updated